net/pfSense-pkg-haproxy-devel/files/usr/local/www/shortcuts/pkg_haproxy.inc

```diff
@@ -24,6 +24,6 @@ global $shortcuts;
 
 $shortcuts['haproxy'] = array();
 $shortcuts['haproxy']['main'] = "/haproxy/haproxy_global.php";
-$shortcuts['haproxy']['log'] = "/status_pkglogs.php?pkg=haproxy-devel";
+$shortcuts['haproxy']['log'] = "/status_logs_packages.php?pkg=haproxy-devel";
```

Comment:
The old link is correct on 2.4.x, the new link is correct on 2.5.x.

Also needs a version bump.